Arelle/Arelle
# Add XBRL 2.1 Conformance Test suite to CI

#### Reason for change
Increase conformance suite coverage to include XBRL 2.1 specification

#### Description of change
Add python test and run script for suite and add to Github actions

#### Steps to Test
Run `run_xbrl_conformance_suite.py` and ensure tests complete.
Due to `pull_request_target` we can't run actions as updated in our PRs, so we can only test that portion after merging.

**review**:
@Arelle/arelle
